lib/beaker/dsl/helpers/host_helpers.rb

```diff
@@ -606,6 +606,31 @@ def echo_on hosts, val
             end
           end
         end
+
+        def ruby_command(host)
+          "env PATH=\"#{host['privatebindir']}:${PATH}\" ruby"
```

Comment:
@jsane `host['privatebindir']` won't be available necessarily to beaker in the next major version (coming soon), as it will only be getting set from beaker-puppet or beaker-pe. Since that's the case, I don't believe beaker is the right place for these methods.

beaker-pe depends on beaker-puppet though, so I believe these methods should go into that repo. Let me know if you have any questions about this.